pkg/app/metadata.json

```
@@ -0,0 +1,959 @@
+        "title": "Deploy a package to a Fastly Compute@Edge service"
+      },
+      {
+        "cmd": "fastly compute deploy --package ./pkg/example.tar.gz",
+        "description": "Use the `compute pack` command to package up a pre-compiled Wasm binary and then reference the generated archive file when deploying.",
+        "title": "Deploy a custom package to a Fastly Compute@Edge service"
+      }],
+      "apis": [
+        "https://developer.fastly.com/reference/api/services/service/#create-service",
+        "https://developer.fastly.com/reference/api/services/service/#get-service",
+        "https://developer.fastly.com/reference/api/services/package/#get-package",
+        "https://developer.fastly.com/reference/api/services/package/#put-package"
+      ]
+    },
+    "init": {
+      "examples": [{
+        "cmd": "fastly compute init --name example --language rust",
+        "description": "To initialize a new Compute@Edge package you must select a supported language. The language can be provided using the optional `--language` flag, which supports tab completion hints, or the flag can be omitted and you'll be prompted interactively. The `--name` flag can also be omitted, which will result in the CLI prompting you interactively.",
+        "title": "Initialize a new Compute@Edge package locally"
+      },
+      {
+        "cmd": "fastly compute init --from=https://fiddle.fastlydemo.net/fiddle/0220c0d2",
+        "description": "Any [Compute@Edge examples](https://developer.fastly.com/solutions/examples/) can be used as a source template for your new package.",
+        "title": "Initialize a new Compute@Edge package locally using a remote package template"
+      },
+      {
+        "cmd": "fastly compute init --directory ./example",
+        "description": "It's recommended you change to the new project directory before executing further CLI commands.",
+        "title": "Initialize a new Compute@Edge package locally in a different directory"
+      }]
+    },
+    "pack": {
+      "examples": [{
+        "cmd": "fastly compute pack --wasm-binary ./bin/main.wasm",
+        "description": "Write Compute@Edge applications in [any WASI-supporting language](https://developer.fastly.com/learning/compute/custom/) and use `compute pack` to package the pre-compiled Wasm binary into a supported format.",
+        "title": "Package a pre-compiled Wasm binary for a Fastly Compute@Edge service"
+      }]
+    },
+    "publish": {
+      "examples": [{
+        "cmd": "fastly compute publish --skip-verification --accept-defaults",
+        "description": "The `compute publish` command is a convenience wrapper around the existing build and deploy commands. All flags present on the `compute build` and `compute deploy` commands are available to use here.",
```

It actually occurs to me that in order to crossref CLI commands they need to be KBD:
```suggestion
        "description": "The <kbd>fastly compute publish</kbd> command is a convenience wrapper around the existing build and deploy commands. All flags present on the `compute build` and `compute deploy` commands are available to use here.",
```
I'm ambivalent about changing this now, but we may want to consider it at some point

---

OK I'll double check any other places where we need to cross-reference commands.